### ISSUE TITLE
ortholog identifiers

### ISSUE THREAD

**Issue by selewis** [monarch-initiative/biolink-api#66]
This may turn into 2-3 separate issues, but lets start. The objective is to determine whether any of the orthologs of a given gene have been functionally described using GO.

This query: 
https://api.monarchinitiative.org/api/bioentity/gene/MGI%3A97490/homologs/?homology_type=O&rows=20&fetch_objects=true

Returns these results:
    "FlyBase:FBgn0005558",
    "FlyBase:FBgn0019650",
    "NCBIGene:100022926",
    "NCBIGene:100514152",
    "NCBIGene:181251",
    "NCBIGene:25509",
    "NCBIGene:286857",
    "NCBIGene:5080",
    "NCBIGene:695746",
    "NCBIGene:737387",
    "ZFIN:ZDB-GENE-041210-244",
    "ZFIN:ZDB-GENE-081022-10",
    "ZFIN:ZDB-GENE-990415-200"

Respectively: fly, fly, possum, pig, WORM, RAT, bovine, HUMAN, macaque, chimp, zfish, zfish, zfish
1. First problem is that my objective was to use these results to issue a subsequent query to determine whether these homologs have experimental GO annotations associated with them. However AmiGO/GOlr doesn't recognize NCBIGene identifiers so I'll never see results that are actually available. The is especially harmful for human genes. Need the UniProt identifier to find results in AmiGO.

2.  There is also a worm ortholog that is not showing the WB id (NCBIGene:181251 is worm). Same for rat (NCBIGene:25509), need the RGD identifier.

**Comment by cmungall** [comment]
some context: the monarch biolink is configured to query from the go golr for any function queries, we we annotate humans to UniProt GCRPs. These are not in the equivalence clique since proteins != genes.

@kshefchek I thought I made a ticket about this but I imagined it? I also thought I put this in the graphstore reasoning doc but it seems not. The most elegant way to handle this is to make equivalent-as cliques or to propagate/materialize in scigraph...

but for now, I think we can encode the inference in the biolink call?
